### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by cowan-macady**
implementing ENG-2648

### CODE REVIEW

File: indykite_sdk/usage/knowledge.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
+"""
```
Comment (kilimandzsaro) [inline indykite_sdk/usage/knowledge.py:1]: I suggest to remove all the "DigitalTwin" reference from here as well.

**Review by kilimandzsaro — CHANGES_REQUESTED** [review]
@cowan-macady We are trying to get rid of the "DigitalTwin" concept and the Ingest and Knowledge packages are not using them anymore. I suggest to remove all "DigitalTwin" reference from both name and comments from everywhere. I suggest to use the "IdentityNode" instead of that (if the node is an identity node) or just the "Node" if it's not an identity one.

File: indykite_sdk/usage/authorization.py [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    is_authorized_dt_parser.add_argument("digital_twin_id", help="Digital Twin id (gid)")
+
+    # is_authorized_token
+    is_authorized_token_parser = subparsers.add_parser("is_authorized_token")
+    is_authorized_token_parser.add_argument("access_token")
+
+    # is_authorized_property
+    is_authorized_property_parser = subparsers.add_parser("is_authorized_property")
+    is_authorized_property_parser.add_argument("property_type", help="Digital Twin Identity Property")
+    is_authorized_property_parser.add_argument("property_value", help="Digital Twin Identity Property value")
```
[the remark this answers is not in the document]
Comment (cowan-macady) [reply indykite_sdk/usage/authorization.py]: I removed what it is possible so far

File: indykite_sdk/usage/authorization.py [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+    what_authorized_dt_parser = subparsers.add_parser("what_authorized_dt")
+    what_authorized_dt_parser.add_argument("digital_twin_id", help="Digital Twin id (gid)")
+
+    # what_authorized_token
+    what_authorized_token_parser = subparsers.add_parser("what_authorized_token")
+    what_authorized_token_parser.add_argument("access_token")
+
+    # what_authorized_property
+    what_authorized_property_parser = subparsers.add_parser("what_authorized_property")
+    what_authorized_property_parser.add_argument("property_type", help="Digital Twin Identity Property")
```
[the remark this answers is not in the document]
Comment (cowan-macady) [reply indykite_sdk/usage/authorization.py]: I removed what it is possible so far

File: indykite_sdk/usage/authorization.py [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+    what_authorized_property_parser.add_argument("property_value", help="Digital Twin Identity Property value")
```
[the remark this answers is not in the document]
Comment (cowan-macady) [reply indykite_sdk/usage/authorization.py]: I removed what it is possible so far

File: indykite_sdk/usage/ingest.py [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+            type,
+            ["Person"],
+            properties,
+            "",
+            True)
+        # create record with record_id and upsert
+        record = client_ingest.record_upsert(record_id, upsert)
+        print(record)
+        # send the ingestion request and get the response
+        ingest_record_digital_twin = client_ingest.ingest_record(record)
```
[the remark this answers is not in the document]
Comment (cowan-macady) [reply indykite_sdk/usage/ingest.py]: I removed what it is possible so far

File: indykite_sdk/usage/knowledge.py (hunk 1 above)
Comment (cowan-macady) [reply indykite_sdk/usage/knowledge.py:1]: I removed what it is possible so far

**Review by kilimandzsaro — APPROVED** [review]

**Pull request merged**